pnpm-workspaces/packages/two/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@aspect_rules_js//js:defs.bzl", "js_library")
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+
+js_library(
```

Comment:
Since there's no ts can we just put all the src+deps here into `npm_package`?

Reply:
Could maybe do that in other BUILDs as well.

`js_library` really just groups some files together while passing along js + d.ts related metadata. If all we do is then put that into an `npm_package` then I don't think it's necessary.

Reply:
`npm_package` doesn't have a `deps` attributed. Where do I put the deps from `js_library`?